src/redux/actions/auth.js

```diff
@@ -0,0 +1,56 @@
+import ACTIONS from '../actionTypes'
+
+const signIn = credentials => {
+  return (dispatch, getState, { getFirebase }) => {
+    const firebase = getFirebase()
+    firebase
+      .auth()
+      .signInWithEmailAndPassword(credentials.email, credentials.password)
+      .then(() => {
+        dispatch({ type: ACTIONS.LOGIN_SUCCESS })
+      })
+      .catch(err => {
+        dispatch({ type: ACTIONS.LOGIN_ERROR, err })
+      })
+  }
+}
+
+const signOut = () => {
+  return (dispatch, getState, { getFirebase }) => {
+    const firebase = getFirebase()
+    firebase
+      .auth()
+      .signOut()
+      .then(() => {
+        dispatch({ type: ACTIONS.SIGNOUT_SUCCESS })
+      })
+  }
+}
+
+const signUp = newUser => {
```

Comment:
`const signUp = { firstName, lastName, storeName } => {`

Reply:
```const signUp = {email,  password, firstName, lastName, storeName }``` Isnt it too many variables to pass? why don't you want to pass the whole object?